Source/ImageGlass/frmMain.cs

```diff
@@ -350,8 +350,17 @@ private void LoadImages(List<string> imageFilenameList, string filePath)
             // Find the index of current image
             if (filePath.Length > 0)
             {
+
+                // this part of code fixes calls on legacy 8.3 filenames
+                // (for example opening files from IBM Notes)
+                var di = new DirectoryInfo(filePath);
+                filePath = di.FullName;
+
+                var filePathFixed = App.ToAbsolutePath(filePath);
```

Comment:
`filePathFixed`  is not used?

Reply:
hi,

sorry, i forgot about this line of code. filePathFixed should not be there.

Reply:
anyway, thanks!